lib/ttnt/tasks.rb

```diff
@@ -0,0 +1,54 @@
+require 'rake/testtask'
+require 'rake/file_list'
+require 'ttnt/testtask'
+require 'ttnt/test_selector'
+require 'rugged'
+
+# Reference: https://github.com/bundler/bundler/blob/master/lib/bundler/gem_helper.rb
+module TTNT
+  class TaskHelper
+    include Rake::DSL if defined? Rake::DSL
+
+    def self.install_tasks
+      new.install
+    end
+
+    def install
```

Comment:
as mentioned in [above comment](https://github.com/Genki-S/ttnt/pull/6#discussion_r32375977) , I changed this name to `define`.